src/main/java/com/sem/btrouble/game/AbstractGame.java

```diff
@@ -25,7 +22,6 @@
     private LevelObserver view;
     private Timer levelTimer;
 
-    // Max time in seconds.
```

Comment:
This doesn't count as inline comments(I think). Although the name already makes it clear what it's use is.